src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -55,7 +55,9 @@
 _CLIENT_ID = '04b07795-8ddb-461a-bbee-02f9e1bf7b46'
 _COMMON_TENANT = 'common'
 
-_AUTH_CTX_FACTORY = lambda authority, cache: adal.AuthenticationContext(authority, cache=cache)
+_AUTH_CTX_FACTORY = lambda authority, cache: adal.AuthenticationContext(authority,
+                                                                        cache=cache,
+                                                                        api_version=None)
```

Comment:
Setting to `none` is recommended:
https://github.com/AzureAD/azure-activedirectory-library-for-python/blob/dev/adal/authentication_context.py#L68